databases/week2/db1w2hw_hyf_lesson2_hwPart1-part3.sql

```diff
@@ -0,0 +1,128 @@
+-- Part 1: Working with tasks // DDL homework
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO
+  task
+VALUES
+  (
+    37,
```

Comment:
no need to add the id, and it's actually probably a good idea not to add it, just let MySQL auto increment it.